examples/repl/cmdline.rs

```diff
@@ -811,9 +811,10 @@ pub async fn cmdline(context: Context, line: &str, chat_id: &mut ChatId) -> Resu
             ensure!(sel_chat.is_some(), "No chat selected.");
             ensure!(!arg1.is_empty(), "No message text given.");
 
-            let msg = format!("{} {}", arg1, arg2);
-
-            chat::send_text_msg(&context, sel_chat.as_ref().unwrap().get_id(), msg).await?;
+            let mut msg = Message::new(Viewtype::Text);
+            msg.set_text(Some(format!("{} {}", arg1, arg2)));
+            msg.set_override_sender_name(Some("fluppy".to_string()));
```

Comment:
This makes all messages sent by REPL to be sent as "fluppy"

Reply:
sorry, that was a short test that sneaked in, i removed these changes.